src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -105,6 +105,15 @@ public override DatabaseModel Create(DbConnection dbConnection, DatabaseModelFac
 
             try
             {
+                var internalSchemaList = new List<string>{ "pg_catalog", "information_schema" };
+                using (var command = new NpgsqlCommand("SELECT version()", connection))
+                {
+                    var longVersion = (string)command.ExecuteScalar();
+                    if (longVersion.Contains("CockroachDB"))
+                        internalSchemaList.Add("crdb_internal");
+                }
+                var internalSchemas = string.Join(", ", internalSchemaList.Select(s => $"'{s}'"));
```

Comment:
Let's add quotes to literals and only join strings here without any LINQ.

Reply:
Why?

Reply:
Because the original strings are literals and not fetched from somewhere. So you can just surround them with single quotes and remove `Select` call.

Reply:
OK, I'll make the change to make you happy :)